test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -320,8 +320,10 @@ test
     await t.expect(saveBtnText).eql('Verify');
 
     const emailAddress = emailVerification.currentAuthenticatorEnrollment.value.profile.email;
+    const enterVerificationCodeText = challengeEmailPageObject.getEnterVerificationCodeText();
     await t.expect(challengeEmailPageObject.getFormSubtitle())
       .eql(`We sent an email to ${emailAddress}. Click the verification link in your email to continue or enter the code below.`);
+    await t.expect(enterVerificationCodeText).eql('Enter a verification code instead');
```

Comment:
nit: Make the string `'Enter a verification code instead'` a constant so you can reuse it within tests

Reply:
done